setup.cfg

```
@@ -39,7 +39,7 @@ install_requires =
     django-filter>=2.0
     django-solo
     djangorestframework~=3.12.0
-    djangorestframework_camel_case
+    djangorestframework_camel_case~=1.2.0
```

```suggestion
    djangorestframework_camel_case>=1.2.0
```

I don't see why a future 1.3.0 shouldn't work if they stick to semver